docker/checkv/Dockerfile

```diff
@@ -12,7 +12,7 @@ LABEL software.version="0.8.1"
 ENV VERSION 0.8.1
```

Comment:
we could also bump all tool versions at some point. E.g. `checkv` is at 1.0.1 already. Not necessarily now, but maybe in another update

Reply:
Captured in https://github.com/EBI-Metagenomics/emg-viral-pipeline/issues/86